src/api/l_graphics.c

```diff
@@ -1062,7 +1062,7 @@ static int l_lovrGraphicsNewShader(lua_State* L) {
   }
 
   info.flags = flags.data;
-  info.flagCount = flags.length;
+  info.flagCount = (graphics_size)(flags.length);
```

Comment:
Note that flagCount is not a size, it's a count.  It does need to get narrowed to a u32 at some point, but it should not use `graphics_size` since it does not represent the size or offset of a piece of GPU memory.